app-web/registry/featured-cards.json

```diff
@@ -11,6 +11,18 @@
               "url": "https://signing-web-devhub-prod.pathfinder.gov.bc.ca"
             }
           },
+          {
+            "sourceType": "github",
+            "resourceType": "Documentation",
+            "sourceProperties": {
+              "url": "https://github.com/bcgov/api-guidelines",
+              "owner": "bcgov",
+              "repo": "api-guidelines",
+              "files": [
+                "government-of-british-columbia-api-guidelines.md"
```

Comment:
If you could change the title of your file so that they match. It will make the user experience when navigating to this content easier. 

![Uploading Screen Shot 2019-06-05 at 9.27.08 AM.png…]()

Reply:
<img width="959" alt="Screen Shot 2019-06-05 at 9 27 08 AM" src="https://user-images.githubusercontent.com/21046727/58973231-4e3baa80-8774-11e9-82ef-63e795aa19a8.png">

Reply:
`title: BC Government API Guidelines` and __Government of British Columbia API Guidelines DRAFT__

Reply:
updated.